docs/src/model_structure.md

```diff
@@ -0,0 +1,38 @@
+# ClimaLand model structure
+
+Here, we explain the internal organization of ClimaLand models.
+
+## Standalone and integrated models
+A core concept of ClimaLand's design is the presence of _standalone_ and _integrated_ models.
+In ClimaLand's framework, the land system is divided into individual components: soil, vegetation (or canopy), snow, and surface water (or pond). Each of these components can be run individually (_standalone_) or combined together (_integrated_).
```

Comment:
Let's remove the pond one? That is really just used in a demo, but not in any of our core models yet

Reply:
Sounds good to me. I also left out SoilCO2 since I don't think we have any standalone examples with it.

Reply:
sounds good!